tuna/db/tuna_tables.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+""" Module for creating DB tables"""
+import enum
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy import Text, Enum
+from sqlalchemy.ext.declarative import declared_attr
+
+
+#pylint: disable=too-few-public-methods
+class JobEnum(enum.Enum):
```

Comment:
Remove the copy of this from miopen_tables, and import the enum from db/tuna_tables to miopen/db/miopen_tables.

Reply:
we use state = Column(Enum(JobEnum), nullable=False, server_default="new") in JobMixin class. Which is part of tuna/db/tuna_tables.py , with new folder and file creation